src/xdocs/config_design.xml

```diff
@@ -289,39 +289,43 @@
                     Example:
                 </p>
                 <p>
-                <source>
-                    final class MyClass{
-                    private MyClass(){ } //OK
-                    }
-                    class MyClass(){
-                    private MyClass() { }
-                    } //violation,class should be declared final
-
-                    class MyClass{
-                    int a,String str;
-                    private MyClass(int a){
-                    this.a=a;
-                    str="dummy value";
-                    }
-                    public MyClass(String str){
-                    this.str=str;
-                    a=0;
-                    } //OK,since it has a public constructor
-                    }
+                    <source>
+                        final class MyClass{
+                        private MyClass(){ } //OK
+                        }
+                        class MyClass(){
```

Comment:
Please remove the parenthesis in the class. It is causing the file not to build.